package.json

```diff
@@ -131,5 +131,8 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "config": {
+    "chromeDriver": "78.0.3904.70"
```

Comment:
can you try to use a more recent one? like maybe 80.0.3987.0

Reply:
Yes, that is the second step if the initial config works